src/models/topic.test.ts

```diff
@@ -133,6 +133,7 @@ describe("KafkaTopicTreeItem constructor", () => {
     const icon = noSchemaTopicTreeItem.iconPath;
     assert.strictEqual(icon instanceof vscode.ThemeIcon, true);
     assert.strictEqual((icon as vscode.ThemeIcon).id, IconNames.TOPIC_WITHOUT_SCHEMA);
+    // @ts-expect-error we need to update @types/vscode to make ThemeColor.id public
     assert.strictEqual((icon as vscode.ThemeIcon).color!.id, "problemsWarningIcon.foreground");
```

Comment:
Unrelated to this PR: later versions of `@types/vscode` provide `id`: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ab4715a8aead1731ced56619128c31e57bcc682d/types/vscode/index.d.ts#L904-L909